seed.js

```diff
@@ -1,9 +1,38 @@
 // This file allows us to seed our application with data
 // simply run: `node seed.js` from the root of this project folder.
 
-// var db = require('./models');
+var db = require('./models');
 
 // var new_campsite = {description: "Sharp rocks. Middle of nowhere."}
+let superHero = [
+  {
+    name: "Spider-Man",
+    powers: "Spider Things",
+    age: 23,
+    secret_identity: true
+  },
+  {
+    name: "Batman",
+    powers: "Unbreakable Will",
+    age: 32,
+    secret_identity: true
+  },
+  {
+    name: "Thor",
+    powers: "Thunder God abilities",
+    age: 1500,
+    secret_identity: false
+  }
+]
+
+// superHero.forEach(function(new_hero){
```

Comment:
This code looks helpful! Sad that it's commented out, so your seed file doesn't get to do anything.